ECE-Tools 2002.1.3 docs and release notes (#8161)
magento/devdocs: src/cloud/env/variables-build.md

    SKIP_SCD: false
```

> ### `SKIP_COMPOSER_DUMP_AUTOLOAD`

Reviewer: The variables are in alphabetical order. This section should be before SKIP_SCD.

> Set to `true` to skip the `composer dump-autoload` command during a {{ site.data.var.mcd-prod }} installation. This variable is only relevant for {{ site.data.var.mcd-prod }}  containers with writable file systems. In such cases, skipping the command prevents errors from other commands trying to access code from the deleted `generated` directory.

Reviewer: ```suggestion
Set to `true` to skip the `composer dump-autoload` command during a {{ site.data.var.mcd-prod }} installation. This variable is only relevant for {{ site.data.var.mcd-prod }} containers with writable file systems. In such cases, skipping the command prevents errors from other commands trying to access code from the deleted `generated` directory.
```

> When Magento runs `composer dump-autoload`, it creates autoload files with links to generated classes in the `generated` folder. In production environments with read-only files systems, this is not a problem. But for Cloud Docker installations with writable file systems, you can run the `bin/magento -n setup:upgrade` command without the `--keep-generated` option, which deletes the `generated` directory. At this point, the next command fails because autoload contains links to deleted files.

Reviewer 2: Consider:  Not sure if we need to include the second detailed explanation. Is it enough for the user to know that they only need to use the variable when deploying a Cloud Docker environment with writable file systems?

```suggestion
When Magento runs `composer dump-autoload`, it creates autoload files with links to generated classes in the `generated` folder. In production environments with read-only files systems, this is not a problem. However, for {{ site.data.var.mcd-prod }} installations with writable file systems, you can run the `bin/magento -n setup:upgrade` command without the `--keep-generated` option, which deletes the `generated` directory. After that, the `composer dump autoload` command fails because the autoload contains links to files in the deleted directory.
```

> When Magento runs `composer dump-autoload`, it creates autoload files with links to generated classes in the `generated` folder. In production environments with read-only files systems, this is not a problem. However, for {{ site.data.var.mcd-prod }} installations with writable file systems, you can run the `bin/magento -n setup:upgrade` command without the `--keep-generated` option, which deletes the `generated` directory. After that, the `composer dump autoload` command fails because the autoload contains links to files in the deleted directory.

Reviewer: All remote environments are read-only: Production, Staging, and Integration. With Docker, it's the developer mode? Also, need to add or remove a hyphen for the `composer dump autoload` command, to be consistent. There are two here.

"After that" be more specific about "that" or remove it.

> -  **Default**— _Not set_

Reviewer: The default and version line need to go after the heading for consistency.

Author: Thanks, Heather. I'm glad you looked at those doc changes. I was too focused on the release notes and forgot to edit them myself. Oops and cheers!

-  **Version**—Magento 2.1.4 and later

```yaml

> stage:
>    build:
>     SKIP_COMPOSER_DUMP_AUTOLOAD: true

Reviewer: ```suggestion
stage:
  build:
    SKIP_COMPOSER_DUMP_AUTOLOAD: true
```
